src/java.base/share/classes/java/util/Optional.java

```diff
@@ -428,6 +431,7 @@ public T orElseThrow() {
      * @throws NullPointerException if no value is present and the exception
      *          supplying function is {@code null}
      */
+    @EnsuresQualifier(expression = "this", qualifier = Present.class)
```

Comment:
The only effect of the two `@EnsuresQualifier` annotations is to restrict the Optional Checker to issue just one error, at the first problem, and not to issue an error on a subsequent line.  I assume that is your intention.

Reply:
I actually thought the the first problem was a false positive, because I did not know that it was actually intended.
In that case this line would have made an actual difference.

But as it doesn't hurt and has same small advantage let's keep it in.